README.md

```diff
@@ -43,6 +43,7 @@ class MyComponent extends Component {
 
 ###### Available Options (props)
 * **date:** *(String, Moment.js object, Function)* - default: today
+* **disableDaysBeforeDate:** *(String)* - A date string in the format "YYYY-MM-DD"
```

Comment:
Just curious, why don't we simply use `minDate` prop here? Doesn't it satisfy our needs?